app/services/spree_shopify_importer/data_parsers/zones/base_data.rb

```diff
@@ -0,0 +1,52 @@
+module SpreeShopifyImporter
+  module DataParsers
+    module Zones
+      class BaseData
+        def initialize(shopify_shipping_zone)
+          @shopify_shipping_zone = shopify_shipping_zone
+        end
+
+        def attributes
+          @attributes ||= {
+            name: @shopify_shipping_zone.name,
+            description: description,
+            kind: @kind
```

Comment:
`kind` of what?
not very clear name, to be honest

Reply:
it is just the attribute of zone, no my idea

Reply:
if you know what is this attribute means, you can just change the name of your `@kind` variable.
but, for sure, you can leave it as is.

Reply:
why should I change name of variable? it keeps value of this attribute, it is not the best way of naming?

Reply:
I just mean, for example
`kind: @kind_of_something`, just more precise name, because for me when I see in the code I don't understand what's the `kind` and I need to spend a lot of time to get it.
but if you want you can leave it as it